app/src/main/kotlin/io/homeassistant/companion/android/webview/WebViewContentScreen.kt

```diff
@@ -58,18 +81,21 @@ internal fun WebViewContentScreen(
     playerLeft: Dp,
     currentAppLocked: Boolean,
     customViewFromWebView: View?,
+    shouldAskNotificationPermission: Boolean,
+    webViewInitialized: Boolean,
+    onFullscreenClicked: (isFullscreen: Boolean) -> Unit,
+    onDiscardNotificationPermission: () -> Unit,
     nightModeTheme: NightModeTheme? = null,
     statusBarColor: Color? = null,
     backgroundColor: Color? = null,
-    onFullscreenClicked: (isFullscreen: Boolean) -> Unit,
+    hasNotificationPermission: Boolean = Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU,
```

Comment:
`has` can also mean the permission has been granted to the app

```suggestion
    supportsNotificationPermission: Boolean = Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU,
```